This is not a proper fix, but a workaround that is needed on HEAD.  (Have
to check glibc-2_5-branch.)  Reported on
<http://savannah.gnu.org/bugs/?18218>.

We need to keep this until a newer hurd package gets compiled with MSG_NOSIGNAL
support (>= 20080921).

---
 resolv/res_send.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/resolv/res_send.c
+++ b/resolv/res_send.c
@@ -174,6 +174,8 @@
 		TIMEVAL_TO_TIMESPEC (&now, res);
 }
 
+#undef MSG_NOSIGNAL
+#define MSG_NOSIGNAL	0
 
 /* Options.  Leave them on. */
 /* #undef DEBUG */
